tables.md

```diff
@@ -0,0 +1,25 @@
+# Tables Pre-calculation
```

Comment:
seems that this file is repeated:
- `tables.md`
- `prover/tables.md`

Reply:
Thanks

Reply:
I creating a mess of the repo with rebase.